### PR TITLE
Fix some nits on psap-configuring-node-feature-discovery.adoc

### PR DESCRIPTION
**Pull request by leo8a**
Signed-off-by: Leo Ochoa <lochoa@redhat.com>

### CODE REVIEW

**Comment by leo8a** [comment]
It probably needs to be cherrypicked in enterprise-4.8/9 as well.

File: modules/psap-configuring-node-feature-discovery.adoc [hunk 1]
```diff
@@ -287,9 +287,9 @@ sources:
 
 [discrete]
 [id="configuring-node-feature-discovery-operator-sources-pci-deviceClassWhitelist_{context}"]
-=== soures.pci.deviceClassWhitelist
+=== sources.pci.deviceClassWhitelist
```
Comment (leo8a) [inline modules/psap-configuring-node-feature-discovery.adoc:290]: Basically, I just s/soures/sources throughout this page.

**Review by ahardin-rh — APPROVED** [review]
LGTM! Thank you!

**Comment by ahardin-rh** [comment]
cherrypick enterprise-4.9

**Comment by ahardin-rh** [comment]
cherrypick enterprise-4.8

**Pull request merged**